From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Loki Rautio <lokirautio@gmail.com>
Date: Sun, 22 Oct 2023 15:15:09 -0500
Subject: [PATCH] fully remove isChunkTracked

apparently was removed upstream so let's just not have it

diff --git a/src/main/java/net/minecraft/server/level/ChunkMap.java b/src/main/java/net/minecraft/server/level/ChunkMap.java
index 8cf0f5c7f91771769ccd199bfb68395fd918ef3c..4a2ddb1322f0f1f18482da6be4b9ab0aea1e81c7 100644
--- a/src/main/java/net/minecraft/server/level/ChunkMap.java
+++ b/src/main/java/net/minecraft/server/level/ChunkMap.java
@@ -1292,7 +1292,7 @@ public class ChunkMap extends ChunkStorage implements ChunkHolder.PlayerProvider
                 double d0 = (double) Math.min(this.getEffectiveRange(), i * 16);
                 double d1 = vec3d_dx * vec3d_dx + vec3d_dz * vec3d_dz;
                 double d2 = d0 * d0;
-                boolean flag = d1 <= d2 && this.entity.broadcastToPlayer(player) && (!level.paperConfig().entities.trackingRangeY.enabled || ChunkMap.this.isChunkTracked(player, this.entity.chunkPosition().x, this.entity.chunkPosition().z));
+                boolean flag = d1 <= d2 && this.entity.broadcastToPlayer(player);// && (!level.paperConfig().entities.trackingRangeY.enabled || ChunkMap.this.isChunkTracked(player, this.entity.chunkPosition().x, this.entity.chunkPosition().z));
                 // Paper start - check Y
                 if (flag && level.paperConfig().entities.trackingRangeY.enabled) {
                     double rangeY = level.paperConfig().entities.trackingRangeY.get(this.entity, -1);
